src/pages/Widget/Configurer/Configurer.tsx

```diff
@@ -55,21 +60,51 @@ export default function Configurer({ classes = "", config, setConfig }: Props) {
         Donation Form Builder
       </h2>
 
-      <div className="grid content-start gap-6 text-sm">
-        <label className="-mb-4">Nonprofit name:</label>
+      <div className="grid content-start text-sm">
```

Comment:
```suggestion
      <div className="grid content-start gap-6 text-sm">
```
remove `gap` and let each child specify spacing around their sibling